src/components/ksi-card.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import logoKSI from '../images/logoKSI.svg';
+import "../styles/ksi-styles.css";
```

Comment:
Please use scss modules 
https://www.gatsbyjs.com/docs/how-to/styling/css-modules/ 
https://www.gatsbyjs.com/plugins/gatsby-plugin-sass#with-css-modules